Source/Scene/Cesium3DTilesetTraversal.js

```diff
@@ -137,9 +137,8 @@ define([
                 tileContent.featurePropertiesDirty = false;
                 tile.lastStyleTime = 0; // Force applying the style to this tile
                 tileset._selectedTilesToStyle.push(tile);
-            } else if ((tile._selectedFrame !== frameState.frameNumber - 1)) {
+            } else if ((tile._selectedFrame < frameState.frameNumber - 1)) {
```

Comment:
This caused the second bug. It wasn't accounting for the fact that the pick and render pass are the same frame number.